homeassistant/components/jewish_calendar/sensor.py

```diff
@@ -250,11 +258,11 @@ class JewishCalendarTimeSensor(JewishCalendarSensor):
     _attr_device_class = DEVICE_CLASS_TIMESTAMP
 
     @property
-    def native_value(self):
+    def native_value(self) -> StateType | None:
         """Return the state of the sensor."""
         if self._state is None:
             return None
-        return dt_util.as_utc(self._state).isoformat()
+        return dt_util.as_utc(self._state).isoformat()  # type: ignore[unreachable]
```

Comment:
this is not unreachable, there is another problem, like e.g. self._state is not typed.